src/WhatToCook.Application/Services/RecipeServiceQuery.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.EntityFrameworkCore;
+using WhatToCook.Application.Domain;
+using WhatToCook.Application.Infrastructure;
+using WhatToCook.WebApp.DataTransferObject.Responses;
+
+namespace WhatToCook.Application.Services
+{
+    public class RecipeServiceQuery
+    {
+        private readonly DatabaseContext _dbcontext;
+
+        public RecipeServiceQuery(DatabaseContext dbcontext)
+        {
+            _dbcontext = dbcontext;
+        }
+
+        public async Task<List<RecipeResponse>> GetRecipes()
+        {
+            List<Recipe> recipes = await _dbcontext.Recipes.ToListAsync(); //== //select * from Recipes
```

Comment:
Query could be improved to do mapping directly on EF Core side, see class `MealPlanningServiceQuery`

Reply:
done, will be in the next commit